src/porepy/numerics/mixed_dim/pressure_trace.py

```diff
@@ -0,0 +1,186 @@
+"""
+The module contains the class that implement pressure trace discretization,
+no PDE are associated with this discretization.
+"""
+
+import numpy as np
+import scipy.sparse as sps
+
+import porepy as pp
+
+
+class PressureTrace(pp.Assembler):
+    """ Implement a pressure trace discretization, no PDE are associated with this discretization.
+    Only the interface operator are given. A possible usage is for the co-dimensional objects in a
```

Comment:
I don't understand the purpose here. You want to map cell center pressures to the boundary? Why do you need a separate discretization for this? And which numerical method are you assuming? It will not be consistent with a FV approach.

Reply:
It does not capture the pressure difference from cell center to the face caused by the flux over the face. The point is, the pressure trace operator is already defined for all EllipticDiscretization methods, this is what is used in mixed-dimensional couplings.

Reply:
the PressureTrace class is like the classes in the files fv_elliptic.py or dual_elliptic.py. The thing is, I need a class that realise no-equation in the domain (the matrix and rhs are empty) but give some coupling conditions for the higher dimensional domain. This is an approach similar to what we have used in the DFN paper we did. The variable in the lower dimensional domain can be used as Lagrange multiplier to couple the flow. The name might be misleading, thou

Reply:
I think I understand. The name is misleading, and the documentation must be improved.

Reply:
Sure, I'll do it